examples/channels-plugin-example/channel/scripts/create-channel-accounts.ts

```diff
@@ -0,0 +1,1022 @@
+import { Buffer } from 'node:buffer';
```

Comment:
Does it makes sense to move this script to plugin itself and just reference it from README?